packages/@o3r/localization/src/devkit/localization-devkit.interface.ts

```diff
@@ -20,3 +28,46 @@ export type AvailableLocalizationMessageContents =
   | LocalizationMessageContents
   | ConnectContentMessage
   | RequestMessagesContentMessage<LocalizationMessageDataTypes>;
+
+
+/**
+ * Contextualization devtools exposed for localization in CMS integration
+ */
+export interface LocalizationContextualizationDevtools {
+  /**
+   * Show localization keys
+   * @param value value enforced by the DevTools extension
+   */
+  showLocalizationKeys: (value?: boolean) => Promise<void>;
```

Comment:
Could be `void | Promise<void>` (same for others) to avoid the `return Promise.resolve(....);` on function with return value